src/content/documentation/publish/self-distribution.md

```diff
@@ -61,6 +61,52 @@ When you choose to distribute an add-on yourself, they can be installed using th
   content: content
 %}
 
+{% capture content %}
+
+### Web download
+
+When distributing an add–on from your website, you have two options for making the signed add–on file available for download:
+
+- From a link on a webpage, like this:
+  ```html
+  <div id="example-option-1" class="install-ok">
+    <a href="https://example.com/path/to/extension.xpi">
+      Install my add-on
+    </a>
+  </div>
+  ```
+  This is the recommended approach.
+- Using JavaScript, like this:
+  ```html
+  <div id="example-option-2" class="install-ok">
+    <button>
+      Install my add-on
+    </button>
+    <script>
+      document.querySelector("#example-option-2 > button").onclick = () => {
+        window.location = "https://example.com/path/to/extension.xpi";
+      };
+    </script>
+  </div>
+  ```
+  If you take this approach, make sure that:
+  - The browser detects your JavaScript code as handling user input, for example, it is called from inside a DOM click event.
+  - If the installation is not triggered from the top-level frame of the website, none of the frames between the top-level page and the frame triggering the installation are a third party to the top-level page.
```

Comment:
Not having any cross origin frames in between is actually a requirement also for installations triggered without any website JS code involved (and so also for a plain link to the xpi url).

Reply:
Updated, please check I've understood your feedback correctly

Reply:
@rebloor thanks, :ok_hand:  yes the last change seems to be handling what I was pointing out, just one question:
- while looking to the interdiff I did notice that the same restriction is described with a slightly different wording in the two described approaches, any reason to don't use the exact same wording? 
(and eventually move that outside of the two approaches and into a note box as for the part that mention the deprecation of the non-standard InstallTrigger Web API).